### PR TITLE
Wait for elastic index after experiment creation in trashing e2e

### PR DESCRIPTION
**Pull request by jdudycz**
After switching to new elasticsearch client code in leaderboard it seems that we need to wait a bit for index refresh after creating test experiments

### CODE REVIEW

File: tests/e2e/management/test_management.py [hunk 1]
```diff
@@ -415,6 +415,8 @@ def test_trash_runs_and_models(self, project, environment):
         run2_id = initialize_container(ContainerType.RUN, project=environment.project)["sys/id"].fetch()
         model1_id = initialize_container(ContainerType.MODEL, project=environment.project)["sys/id"].fetch()
         model2_id = initialize_container(ContainerType.MODEL, project=environment.project)["sys/id"].fetch()
+        # wait for the elasticsearch cache to fill
+        sleep(5)
```
Comment (aniezurawski) [inline tests/e2e/management/test_management.py:419]: @Raalsky what do you think about creating some `eventualy(predicate, timeout)` method with expotentional retries?

Comment (jdudycz) [reply tests/e2e/management/test_management.py:419]: I've created a task for that https://neptune-labs.atlassian.net/browse/NPT-13399

Comment (Raalsky) [reply tests/e2e/management/test_management.py:419]: I've already been using `backoff` package in a few integrations tests. Something like:
```python
@backoff.on_exception(backoff.expo, AssertionError, max_value=1, max_time=60)
def test_....():
    ....
```

Comment (Raalsky) [reply tests/e2e/management/test_management.py:419]: I think that there is something like a constant for the number of seconds to wait.

Comment (jdudycz) [reply tests/e2e/management/test_management.py:419]: Okay, will do

Comment (jdudycz) [reply tests/e2e/management/test_management.py]: Done

**Review by aniezurawski — APPROVED** [review]

**Pull request merged**